apier/v1/tpsars.go

```diff
@@ -0,0 +1,101 @@
+/*
+Real-time Online/Offline Charging System (OCS) for Telecom & ISP environments
+Copyright (C) ITsysCOM GmbH
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>
+*/
+
+package v1
+
+import (
+	"github.com/cgrates/birpc/context"
+	"github.com/cgrates/cgrates/utils"
+)
+
+// SetTPSags creates a new stat within a tariff plan
+func (apierSv1 *APIerSv1) SetTPSar(ctx *context.Context, sar *utils.TPSarsProfile, reply *string) error {
+	if missing := utils.MissingStructFields(sar, []string{utils.TPid, utils.ID}); len(missing) != 0 {
+		return utils.NewErrMandatoryIeMissing(missing...)
+	}
+	if sar.Tenant == utils.EmptyString {
+		sar.Tenant = apierSv1.Config.GeneralCfg().DefaultTenant
+	}
+	if err := apierSv1.StorDb.SetTPSars([]*utils.TPSarsProfile{sar}); err != nil {
+		return utils.APIErrorHandler(err)
+	}
+	*reply = utils.OK
+	return nil
+}
+
+// GetTPSag queries specific Stat on Tariff plan
```

Comment:
Please review all comments and improve. Each API call should have it's documentation attached in the comment.